newscoop/library/Newscoop/Services/Plugins/ManagerService.php

```diff
@@ -98,7 +109,7 @@ public function installPlugin($pluginName, $version, $output, $notify = true)
             throw new \Exception("Plugin name is invalid, try \"vendor/plugin-name\"", 1);
         }
 
-        $process = new Process('cd ' . $this->newsoopDir . ' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.phar require --no-update ' . $pluginName .':' . $version .' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.phar update ' . $pluginName .'  --prefer-dist --no-dev -n');
+        $process = new Process('cd ' . $this->newsoopDir . ' && php -d memory_limit='.$this->config['internal_memory_limit'].' composer.qphar require ' . $pluginName .':' . $version);
```

Comment:
i wrote it before as composer had weird behaviour at that time with require and only fix was to call update after that. 
:+1: from me.